### PR TITLE
GPII-2616:  Replace `marked` with `markdown-it`...

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [GPII-2616](https://issues.gpii.net/browse/GPII-2616) for details.

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
@sgithens, I have cut a dev release for you to test with your work: `1.1.0-dev.20170927T151006Z.df42535`

The main difference as I mentioned in chat is that HTML is disabled by default, there are instructions  on enabling that in the docs.  Happy to pair on it or look at things async if you like.

**Comment by the-t-in-rtf** [comment]
@amb26, I will shortly begin work on [GPII-2444](https://issues.gpii.net/browse/GPII-2444).  If we can't get to this in the next day or so, I'm afraid we'll need to combine the reviews.

File: src/js/common/md-common.js [hunk 1]
```diff
@@ -28,27 +28,29 @@ gpii.handlebars.helper.md.getMdFunction = function (that) {
...
-
-            return text;
-        });
-    }
-    else {
-        fluid.fail("Could not initialize pagedown converter.  Markdown content will not be parsed.");
-    }
-};
+// TODO: See if this is still needed.
+// gpii.handlebars.helper.md.configureConverter = function (that) {
```
Comment (amb26) [inline src/js/common/md-common.js]: Is it still needed?

File: src/js/common/md-common.js [hunk 2]
```diff
@@ -57,12 +59,13 @@ fluid.defaults("gpii.handlebars.helper.md", {
...
-    },
-    listeners: {
-        "converterAvailable": {
-            funcName: "gpii.handlebars.helper.md.configureConverter",
-            args:     ["{that}"]
-        }
+        "rendererAvailable": null
     }
+    // ,
+    // listeners: {
```
Comment (amb26) [inline src/js/common/md-common.js]: See above

File: src/js/common/md-common.js (hunk 1 above)
Comment (the-t-in-rtf) [reply src/js/common/md-common.js]: Nope, and removed.

File: src/js/common/md-common.js (hunk 2 above)
Comment (the-t-in-rtf) [reply src/js/common/md-common.js]: Removed.

**Comment by the-t-in-rtf** [comment]
@amb26, I've addressed your comments and updated all dependencies as well.

**Comment by the-t-in-rtf** [comment]
Thanks, @amb26.  I cut the release, added release notes, and updated the forward-facing version.

**Pull request merged**